### ISSUE TITLE
job submission isn't set up correctly

### ISSUE THREAD

**Issue by cdkharris** [UCL-ARC/terraform-aws-vslurm#13]
current behavior as of 70d5e5c:
```
[root@ip-10-0-101-104 ansible]# ssh ec2-user@login
Last login: Wed Sep 20 13:39:39 2023 from 10.0.101.104
[ec2-user@vslurm-login ~]$ sinfo
PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
normal*      up 1-00:00:00      1   idle c1
[ec2-user@vslurm-login ~]$ squeue
             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
[ec2-user@vslurm-login ~]$ srun -N 1 -n 1 --pty /bin/bash
[ec2-user@vslurm-login ~]$ srun hostname

^Csrun: Cancelled pending job step with signal 2
srun: error: Unable to create step for job 4: Job/step already completing or completed
[ec2-user@vslurm-login ~]$ exit
exit
srun: error: c1: task 0: Exited with exit code 1
[ec2-user@vslurm-login ~]$ hostname
ip-10-0-101-172.eu-west-2.compute.internal              <---- LOGIN PRIVATE DNS
[ec2-user@vslurm-login ~]$ srun -N 1 -n 1 --pty /bin/bash
[ec2-user@vslurm-login ~]$ hostname
ip-10-0-101-7.eu-west-2.compute.internal                <---- C1 PRIVATE DNS
[ec2-user@vslurm-login ~]$ exit
exit
[ec2-user@vslurm-login ~]$ sinfo
```

expected behavior:

- after submitting a job with `srun -N...` I expected the prompt to indicate that i was working on a compute node (c1)
- i expected `srun hostname` to report either 'c1' or the true aws-style hostname of the compute node; instead it hung on the command until i canceled it

**Comment by cdkharris** [comment]
some ideas:

- there might be an error/typo in the instructions
  - i _was_ confused about the use of `srun` within `srun`.
- this might be a deprecated use of `srun` for this version
- the prompt might be confused because i decorated it with an env variable instead of actually changing the hostname

i'm going to spend some time re-familiarizing myself with `man srun`, node registration, and thinking about better ways to manage the hostnames and their aliases.

it occurs to me that if i could pass `-l` to bash then maybe the propmt would be corrected.

**Comment by cdkharris** [comment]
jobs are pretty much working correctly; i think i was just confused by the prompt. fixed with f72ccf14e6021542b89d1b8499ae6bf9d0e29d04
